metadata/hinge.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<wayfire>
+  <plugin name="hinge">
+    <_short>Hinge</_short>
```

Comment:
It might be a good idea to also include a _long description with the plugin name so it is clear what the plugin does exactly.

Reply:
This is what I've written up: 

Given a filename, this plugin reads an int32 every 'poll_freq' milliseconds. This value should be in degrees, so from 0 to 360. If the read value is greater than 'flip_degree' then keyboard and pointer inputs get disabled.

Is this alright?